fastaparse.py

```diff
@@ -0,0 +1,179 @@
+    "UGC": "C",
+    "CGC": "R",
+    "AGC": "S",
+    "GGC": "G",
+    "UGA": "Stop",
+    "CGA": "R",
+    "AGA": "R",
+    "GGA": "G",
+    "UGG": "W",
+    "CGG": "R",
+    "AGG": "R",
+    "GGG": "G"
+}
+
+
+def parse(path):
+    """
+    Function parses a fasta file and returns a dictionary with seq_ids and sequences
+    :param path: path to fasta file
+    :return: dictionary with seq_id as keys and sequence for seq_id as values
+    """
+    with open(path, 'r') as fasta:
+        result = dict()
+        for idx, line in enumerate(fasta):
+            if line.startswith('>'):
+                seq_id = line.strip()[1:]
+                result[seq_id] = ""
+            else:
+                result[seq_id] = f'{result[seq_id]}{line.strip()}'
+    return result
+
+
+def translate(rna_seq):
+    """
+    Function translates input RNA sequence to protein sequence
+    :param rna_seq: string which represents target RNA sequence (it can start from AUG (M) or not,
+    but it must have length multiplied by 3)
+    :return: protein sequence for input RNA sequence
+    """
+    rna_seq = re.sub(" +", '', rna_seq.strip())
+    assert not len(rna_seq) % 3, "Length of RNA sequences must multiple by 3"
+    assert not 'T' in rna_seq, "RNA sequences can't contain Tymin"
+    protein_seq = ""
+    for codon in range(0, len(rna_seq), 3):
+        protein_seq = f'{protein_seq}{codons[rna_seq[codon: codon + 3]]}'
+    return protein_seq
+
+
+def calc_mass(rna_seq):
+    """
+    Function receives RNA sequence as input and returns mass for encoded protein
+    :param rna_seq: string which represents target RNA sequence (it can start from AUG (M) or not,
+    but it must have length multiplied by 3)
+    :return: summary mass for protein encoded by input RNA
+    """
+    rna_seq = re.sub(" +", '', rna_seq.strip())
+    assert not len(rna_seq) % 3, "Length of RNA sequences must multiple by 3"
+    assert not 'T' in rna_seq, "RNA sequences can't contain Tymin"
+    mass = float(0)
+    for codon in range(0, len(rna_seq), 3):
+        aa = codons[rna_seq[codon: codon + 3]]
+        mass += mass_table[aa]
+    return mass
+
+
+def orf(rna_seq):
+    """
+    Function receives RNA sequence as input and returns all possible ORFs
+    If for one start codon there are several stops, the first one will be chosen
+    :param rna_seq: string which represents target RNA sequence (it can start from AUG (M) or not,
+    but it must have length multiplied by 3)
+    :return: all possible ORFs within input RNA sequence (if for one start codon there are several stops,
+    the first one will be chosen)
+    """
+    assert not len(rna_seq) % 3, "Length of RNA sequences must multiple by 3"
+    assert not 'T' in rna_seq, "RNA sequences can't contain Tymin"
+    rna_seq = re.sub(" +", '', rna_seq.strip())
+    starts = list()
+    stops = list()
+    seqs = list()
+
+    for codon in range(0, len(rna_seq) - 2):
+        if codons[rna_seq[codon: codon + 3]] == "M":
```

Comment:
```suggestion
            if codon == "AUG":
```
then we append it to starts